modules/ztp-creating-a-validator-inform-policy.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-creating-a-validator-inform-policy_{context}"]
+= Creating a validator inform policy
+
+Use the following procedure to create a validator inform policy that provides an indication of
+when the zero touch provisioning (ZTP) installation and configuration of the deployed cluster is complete. This policy
+can be used for deployments of single node clusters, three-node clusters, and standard clusters.
+
+.Procedure
+
+. Create a standalone `PolicyGenTemplate` custom resource (CR) that contains the source file
+`validatorCRs/informDuValidator.yaml`.
+You only need one standalone `PolicyGenTemplate` CR for each cluster type.
++
+* Single node clusters:
```

Comment:
Suggestion: These could be three codeblocks with block titles instead of three unordered bullets, if you wanted. E.g.,

```
.Single node clusters
[source,yaml]
----
blah
----
+
.Three-node clusters
[source,yaml]
----
blah
----
+
.Standard clusters
[source,yaml]
----
blah
----
```

Reply:
Looks good, can you remove the colon from the end of the block titles tho?